src/hooks/queries/useIsApprovalRequired.ts

```diff
@@ -10,15 +10,15 @@ import { useAppSelector } from 'state/store'
 
 type UseIsApprovalRequired = {
   assetId: AssetId
-  from: string
+  pubKey: string
```

Comment:
redundant note, but again pref for `from` over `pubkey` for evm only things imo

Reply:
https://github.com/shapeshift/web/commit/d681485bd983a4a4e69cd2b9c3ab493e22cd5427